packages/wdio-types/src/Capabilities.ts

```diff
@@ -39,9 +40,45 @@ export interface ProxyObject {
     noProxy?: string[];
 }
 
+export interface CapabilityOptions extends Pick<TestrunnerOptions, 'specs' | 'exclude' | 'maxInstances'> {
```

Comment:
We don't need this if we define them in the interface with new docs.
```suggestion
export interface CapabilityOptions {
```

Reply:
done